api/routes/image.js

```diff
@@ -62,25 +61,11 @@ router.post('/profile', function(req, res) {
               error: 'Error in saving image urls to user: ' + err
             });
           } else {
-            UserDetails.findOne(
-              {
-                username: user.username
-              },
-              function(err, userDetail) {
-                if (err || !userDetail) {
-                  return res.json({
-                    error: 'Error in retreiving userDetails'
-                  });
-                } else {
-                  return res.json({
-                    user: user,
-                    userDetail: userDetail,
-                    imageURL: req.file.location,
-                    message: 'Successfully saved profile image'
-                  });
-                }
-              }
-            );
+            return res.json({
+              user: user,
```

Comment:
Be careful of what you are returning to the client, this includes sensitive information such as the password.